content/en/ebpf.md

```diff
@@ -0,0 +1,27 @@
+---
+title: eBPF
+status: Completed
+category: architecture
+---
+
+## What it is
+
+Linux system has two spaces: kernel and user space. Kernel space is the core of the operating system and it is the only part that has unlimited access to the hardware. 
+Usually, applications reside in the user space and if they need some higher permissions, they send a request to the kernel space for that.
```

Comment:
```suggestion
Usually, applications reside in the user space, and if they need higher permissions, they send a request to the kernel. 
```
Usually or always? And what does "higher permissions" mean? Can we elaborate?

Reply:
Removed "Usually" and explained the higher permissions.